### PR TITLE
Add cause of death for expired patients

### LINKED ISSUE #1649: Add cause of death to expired patients
When a patient is marked as expired, an additional mandatory field to capture the cause of death must appear.

@bodhish @vigneshhari Please review

### PR DESCRIPTION
**Pull request by Ashesh3**
Fixes #1649

Change "Discharge notes" to "Cause of death" when discharge reason is "Expired". Also, pre-fill the cause on the Death report. 

image
image

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

### CODE REVIEW

File: src/Components/Facility/ConsultationDetails.tsx [hunk 1]
```diff
@@ -402,7 +402,11 @@ export const ConsultationDetails = (props: any) => {
             </div>
 
             <div id="discharge-notes-div">
-              <InputLabel id="refered-label">Discharge Notes</InputLabel>
+              <InputLabel id="refered-label">
+                {preDischargeForm.discharge_reason == "EXP"
+                  ? "Cause of death"
+                  : "Discharge notes"}
+              </InputLabel>
```
Comment (developedBySJ) [inline src/Components/Facility/ConsultationDetails.tsx]: Make the field `mandatory` when discharge reason is `EXP`.
```suggestion
              <InputLabel id="refered-label" required={preDischargeForm.discharge_reason == "EXP"} >
                {preDischargeForm.discharge_reason == "EXP"
                  ? "Cause of death"
                  : "Discharge notes"}
              </InputLabel>
```

Add validations for the same

**Review by developedBySJ — CHANGES_REQUESTED** [review]

File: src/Components/Facility/ConsultationDetails.tsx (hunk 1 above)
Comment (Ashesh3) [reply src/Components/Facility/ConsultationDetails.tsx]: Done.

image

**Comment by nihal467** [comment]
LGTM @khavinshankar review it carefully

**Review by khavinshankar — APPROVED** [review]

**Comment by khavinshankar** [comment]
@developedBySJ can you re-review this

**Pull request merged**